Models/MuZero_trainer.py

```diff
@@ -189,7 +192,9 @@ def get_mean(k):
             tf.summary.scalar('losses/l2', l2_loss, step=train_step)
 
             tf.summary.scalar('accuracy/value', -get_mean('value_diff'), step=train_step)
-            tf.summary.scalar('accuracy/value', -get_mean('reward_diff'), step=train_step)
+            tf.summary.scalar('accuracy/reward', -get_mean('reward_diff'), step=train_step)
```

Comment:
@denguo I think you were the one who did it like this. Are there any complaints about having them separate? It would help to know the model's state if we can adequately see the differences separately.